starters/starters.yaml

```diff
@@ -178,7 +178,7 @@
   description: A minimalistic starter showcasing Shopify & Gridsome working together
   screenshot: ./screenshots/gridsome-starter-shopify.png
   repo: thetre97/gridsome-starter-shopify
-  preview: https://gridsome-shopify-starter.netlify.com
+  preview: https://gridsome-starter-shopify.vercel.app
```

Comment:
Why are we changing this?

Reply:
I simply moved some of my Netlify sites to Vercel, to keep them in one place.
I have left the Netlify site up for now, though.

Reply:
Oh, sorry. I missed that travisreynolds and thetre97 are both you, so I thought you were changing someone else's URL.

Reply:
Long story, but yes, they are both me...